src/Client/components/ques/written-ques.tsx

```
@@ -113,81 +152,79 @@ export default function Component({
         });
     };
     const handlePoints = (questionId: string, value: string) => {
-        const pointsValue = value === "" ? 0 : Math.max(0, parseInt(value) || 0);
+        const pointsValue = value === "" ? 0 : Math.max(0, parseInt(value, 10) || 0);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)